database/src/migrations/20240310163000_observation_sign.ts

```diff
@@ -0,0 +1,104 @@
+import { Knex } from 'knex';
+
+/**
+ * Create new tables with initial data:
+ * - sign
+ *
+ * Update existing tables:
+ * - Add 'sign' column to subcount table
+ *
+ * @export
+ * @param {Knex} knex
+ * @return {*}  {Promise<void>}
+ */
+export async function up(knex: Knex): Promise<void> {
+  await knex.raw(`--sql
+  ----------------------------------------------------------------------------------------
+    -- Create observation subcount sign lookup table
+    ----------------------------------------------------------------------------------------
+    SET search_path = biohub;
+
+    CREATE TABLE observation_subcount_sign (
+      observation_subcount_sign_id                  integer          GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+      name                                          varchar(32)        NOT NULL,
+      description                                   varchar(256),
+      record_end_date                               timestamptz(6),
+      create_date                                   timestamptz(6)     DEFAULT now() NOT NULL,
+      create_user                                   integer            NOT NULL,
+      update_date                                   timestamptz(6),
+      update_user                                   integer,
+      revision_count                                integer            DEFAULT 0 NOT NULL,
+      CONSTRAINT observation_subcount_sign_id_pk PRIMARY KEY (observation_subcount_sign_id)
```

Comment:
super nit: generally the primary key name is based on the table name (so no `_id`):

`CONSTRAINT observation_subcount_sign_pk PRIMARY KEY (observation_subcount_sign_id)`

Doesn't actually hurt anything either way, just the convention we've used so far.

Reply:
Fixed